### PR TITLE
Update security group suggestions

### PR DESCRIPTION
**Pull request by ttheune**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,10 +25,15 @@ for Google Cloud Platform :
...
 * Instantiate an image with the resulting AMI
   * Make sure it has a public IP address
-  * Make sure it is in a security group with the following inbound rules:
+  * Make sure it is in a security group with at least the following inbound rules:
     * UDP 500
     * UDP 4500
     * TCP 22 to your IP (for SSH management)
+  * Make sure it is in a security group with at least the following outbound rules:
+    * UDP 500
+    * UDP 4500
```
Comment (aren) [inline README.md:34]: Seems right but did you test these?

File: README.md [hunk 2]
```diff
@@ -25,10 +25,15 @@ for Google Cloud Platform :
...
+    * TCP 1024-65535 to your IP (for SSH management)
```
Comment (aren) [inline README.md:35]: I don't think this is necessary, but we should test this. Some firewalls are going to allow port 22 outbound if the TCP session was allowed as part of an inbound rule.

File: README.md [hunk 3]
```diff
@@ -25,10 +25,15 @@ for Google Cloud Platform :
...
+    * (optional) Any target ports you want to allow for access
```
Comment (aren) [inline README.md:36]: More detail here -- e.g. "Any target ports your VPN clients might want to connect to. For example, port 443 if you want VPN clients to be able to connect to any HTTPS server."

File: README.md (hunk 1 above)
Comment (ttheune) [reply README.md:34]: I had checked what ports the two sides are listening on, turns out we need ALL UDP

File: README.md (hunk 2 above)
Comment (ttheune) [reply README.md:35]: Need this open to ssh in to the VPN.  Client connects to the VPN on port 22 (inbound) and the VPN connects to the client on the port the client provided which is between 1024-65535 (outbound).

**Pull request closed without merging**